subprojects/packagefiles/lmdb/meson.build

```diff
@@ -0,0 +1,29 @@
+project('lmdb', 'c', version: '0.9.29', license: 'OpenLDAP')
+
+cc = meson.get_compiler('c')
+
+src = [
+  'libraries/liblmdb/mdb.c',
+  'libraries/liblmdb/midl.c',
+]
+
+c_args = [
+  '-W',
+  '-Wall',
+  '-Wno-unused-parameter',
+  '-Wbad-function-cast',
+  '-Wuninitialized'
+]
+
+pthread_dep = cc.find_library('pthread', required: true)
```

Comment:
See https://mesonbuild.com/FAQ.html#what-is-the-correct-way-to-use-threads-such-as-pthreads